interfaces/builtin/basedeclaration.go

```diff
@@ -190,6 +193,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  classic-support:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
```

Comment:
`deny-connection: true`?

We don't want anyone but "classic" using this, whether auto or manual, right?